vendor/github.com/ceph/ceph-csi/api/deploy/kubernetes/nfs/csidriver.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2022 The Ceph-CSI Authors.
```

Comment:
why do we have this vendor change?

Reply:
that is mentioned in the commit message, as `tools/yamlgen` uses the API to generate files under `deploy/`, we are vendoring our own API (I would like to prevent that, but don't know how).